README.md

```diff
@@ -1,57 +1,33 @@
-# Project Name
-
-(short, 1-3 sentenced, description of the project)
-
-## Features
-
-This project framework provides the following features:
-
-* Feature 1
-* Feature 2
-* ...
-
-## Getting Started
-
-### Prerequisites
-
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
-
-### Installation
-
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
-
-### Quickstart
-(Add steps to get up and running quickly)
-
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
-
-
-## Demo
-
-A demo app is included to show how to use the project.
-
-To run the demo, follow these steps:
-
-(Add steps to start up the demo)
-
-1.
-2.
-3.
-
-## Resources
-
-(Any additional resources or related projects)
-
-- Link to supporting information
-- Link to similar sample
-- ...
+# Azure Communication Network Traversal sample
+
+### Prerequisites
+
+You need Node.js
+Firefox, Edge or Chrome
+Computer with camera
+A Communication Services resource, you will need the Connection String.
+
+### Installing
+
+- In a Powershell Terminal go to the sample dir and install the following
+
+npm install
+
+## Setup
+
+- Create a Communication Services resource using your personal subscription. See here for details:

```

Comment:
It says a connection string is a prerequisite but then the setup includes creating a communication service resource